DESCRIPTION

```diff
@@ -34,7 +34,9 @@ Imports:
     tensorflow (>= 1.4.0.9000),
     tfruns (>= 0.9.1),
     tidyselect,
-    utils
+    utils,
+    purrr,
+    dplyr
```

Comment:
I think we can get what you are looking for from dplyr by just importing the `tibble` package (it has row binding and of course as_tibble)

Reply:
I thought `bind_rows()` was `dplyr`-specific?

Reply:
There is definitely row binding code in tibble (I use it in tfruns). You can get as_tibble and glimpse from tibble (http://tibble.tidyverse.org/reference/index.html). Not sure about bind_rows but I wonder if add_row would suffice?

Reply:
Yes, those two functions can be imported from tibble. I tried to use `add_row` but it doesn't seem as efficient as `bind_rows` since the latter is implemented in C++.

Reply:
Okay, sounds good.

Also, we may need to update some of our documentation to reflect `simplify = TRUE` as part of the PR.

Let us know once you think this PR is ready for final review and merge.